src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java

```diff
@@ -58,13 +63,24 @@ default void mapCertifiedCopyItems(final OrderData order,
         List<CertifiedDocument> certifiedDocuments = new ArrayList<>();
         IntStream.range(0, filingHistoryDocuments.size()).forEach(i -> {
             CertifiedDocument certifiedDocument = new CertifiedDocument();
-            certifiedDocument.setDateFiled(filingHistoryDocuments.get(i).getFilingHistoryDate());
+            certifiedDocument.setDateFiled(reformatDateFiled(filingHistoryDocuments.get(i).getFilingHistoryDate()));
             certifiedDocument.setType(filingHistoryDocuments.get(i).getFilingHistoryType());
             certifiedDocument.setDescription(filingHistoryDocuments.get(i).getFilingHistoryDescription());
-            certifiedDocument.setFee("£" + itemCosts.get(i).getCalculatedCost());
+            certifiedDocument.setFee(itemCosts.get(i).getCalculatedCost());
```

Comment:
@VaughanJackson123 Is the currency being assumed and pre-pended on when displaying the results?

Reply:
Yes, as was the case with certificates too. The `£` is in the templates themselves.

Reply:
Approved then 👍